python_modules/dagster/dagster/_core/utils.py

```diff
@@ -114,6 +114,13 @@ def check_dagster_package_version(library_name: str, library_version: str) -> No
             warnings.warn(message)
 
 
+def get_env_var_name(env_var_str: str):
+    if "=" in env_var_str:
+        return env_var_str.split("=", maxsplit=1)[0]
+    else:
+        return env_var_str
```

Comment:
is it a fair expectation that this function could be passed anything other than "KEY=VALUE" strings and handle that case as such?

Reply:
yeah this is intended to handle both KEY=VALUE and KEY (in the latter case it sources VALUE from the calling environment)